### ISSUE TITLE
Top menu nav color and shadow

### ISSUE THREAD

**Issue by damsfx** [wintercms/wn-tailwindui-plugin#5]
https://github.com/wintercms/wn-tailwindui-plugin/blob/cceeac90389a00e262d8af68f85c4fb11fbcbd1c/skins/tailwindui/layouts/_menu-top.php#L13-L17

### 1 - Why switch to a white background?
I find the bar not contrasting enough in the overall interface.

### 2 - For me, the shadow is not visible.

**For it to be visible, a relative position and z-index must be added to the `.layout-topmenu` element.**  
Also I would have chosen a wider shadow (`shadow-md`).

Actual - invisible shadow: 
![image](https://user-images.githubusercontent.com/282242/180041204-d9aa0cd6-b746-451b-93c5-adb50da26ce2.png)

With `shadow` visible : 
![image](https://user-images.githubusercontent.com/282242/180041227-10fb509c-ad02-465b-9ec9-8a8157d47a93.png)

With `shadow-md` visible :
![image](https://user-images.githubusercontent.com/282242/180041257-4641f423-e14b-4476-a14b-204dd436932b.png)

I would be pleased to hear your thoughts on this.

**Comment by LukeTowers** [comment]
@damsfx I would agree with those assessments. As far as colour goes I actually want this plugin to have a light theme and a dark theme with the colour matching between the top bar and the side bar for each version, but for now fixing the shadow to show (the size of it seems fine to me without further changes for now).

**Comment by mjauvin** [comment]
I personally think the top bar serves no actual purpose when the menu is on the left. It breaks the otherwise great look of the backend layout.

**Comment by LukeTowers** [comment]
@mjauvin it doesn't serve much of a purpose yet but it will. The plan is to have a backend-wide search bar in the top left of it as well as a few more additions on the right to the quick actions menu like a quick create button and notification management

**Comment by bennothommo** [comment]
I'm having a play around with this skin for a client at the moment. I can see the rationale for that user bar based on what @LukeTowers wants to do with it going forward, but to offer a counterpoint, the reason that some people prefer side menus (including myself) is the maximise the amount of vertical space for the actual page or function being used, and to take advantage of the amount of width modern monitors have.

Unfortunately, the user bar negates that benefit because it takes up the same amount of space as it would having the main nav at the top, whilst also decreasing the amount of space horizontally too.

See the following:

**Top nav**
![Top Nav Example](https://user-images.githubusercontent.com/15900351/191157417-f298290b-a1ce-4514-b2a5-f409d27f12f3.png)

**Side nav**
![Side Nav Example](https://user-images.githubusercontent.com/15900351/191157408-e7e47935-c02f-4610-90e8-c68324addf9b.png)

Effectively, the usable viewport for the Backend itself is shrunk a little bit if someone uses the side nav option.

Ideally, it would be nice to have the user functions condensed into the side nav, but I can understand if that might be difficult.

**Comment by LukeTowers** [comment]
What I'd like to have is for the side menu layout to have the ability to collapse down to the icon only mode with the press of a button like a mobile menu and then re-show the full size version when hovering over it or clicking the button again.
